### DISCUSSION TITLE
Thoughts on sharing klipper compilation parameters?

### DISCUSSION THREAD

**Discussion by StarkZarn** [bassamanator/Sovol-SV06-firmware#110] [General]
First off, love this repo! I have an SV06 Plus that is an absolute beast now thanks to your work here with Klipper and corresponding configs.

I'm curious though if you'd be open to adding documentation or your .config file for compiling Klipper from source. I'm using kiauh right now as a base framework on my pi, and then flashing your image. The Klipper3d source repo is on 0.12 now and I was thinking about taking a stab at just building my own firmware image.

**Comment by bassamanator** [comment]
Welcome 😀. You might find the everything-sv06 repo useful as well.

```
# During "make menuconfig" select the 
# STM32F103 with a "28KiB bootloader" and serial (on USART1 PA10/PA9) communication.
# Also, since it is using the GD32F103, please select Disable SWD at startup
```

Please star the repo to support this work. Thanks.

**Comment by bassamanator** [comment]
Old save I had around, this should be correct:
![menuconfig-com-serial](https://github.com/bassamanator/Sovol-SV06-firmware/assets/61985779/d8a7039e-8781-48cd-aad2-4e2110d74d9f)

**Comment by bassamanator** [comment]
https://github.com/bassamanator/Sovol-SV06-firmware/discussions/111

**Comment by StarkZarn** [comment]
This is all fantastic, thank you so much! I (stupidly) had not looked through your other repos. You are indeed correct, the everything-sv06 repo will also be a tool for me. I appreciate the quick response. Keep doing what you're doing, cheers!
